beluga/CMakeLists.txt

```diff
@@ -48,6 +56,26 @@ find_package(range-v3 REQUIRED)
 find_package(Sophus REQUIRED)
 find_package(TBB REQUIRED)
 
+# If USE_OPENVDB is enabled, proceed with finding OpenVDB
+if(USE_OPENVDB)
+  if(NOT OPENVDB_CMAKE_MODULE_PATH)
+    find_file(OPENVDB_MODULE_PATH FindOpenVDB.cmake
+              PATHS /usr/lib/x86_64-linux-gnu/cmake/OpenVDB/)
+    if(OPENVDB_MODULE_PATH)
+      get_filename_component(OPENVDB_CMAKE_MODULE_PATH ${OPENVDB_MODULE_PATH}
+                             DIRECTORY)
+    endif()
+  endif()
+  if(OPENVDB_CMAKE_MODULE_PATH)
+    list(APPEND CMAKE_MODULE_PATH ${OPENVDB_CMAKE_MODULE_PATH})
+  endif()
+  find_package(Boost REQUIRED)
```

Comment:
@pvela2017 hmm, do we absolutely need Boost? This doesn't help the OpenVDB as core dependency case.

Reply:
mmm if Boost isnt installed it won't be able to compile. I guess because of this [line](https://github.com/AcademySoftwareFoundation/openvdb/blob/58724e5111b73ae1a1d280e1a65075bf8ad0ef55/openvdb/openvdb/CMakeLists.txt#L112) . I guess we dont need it there but it should be installed on the system

I ran this code to check if `OPENVDB_USE_DELAYED_LOADING` is enabled:
![image](https://github.com/user-attachments/assets/1177e845-36a2-461d-9a39-906436964e88)